dnf/cli/commands/repoquery.py

```diff
@@ -195,7 +192,12 @@ def set_argparser(parser):
                              help=_('show changelogs of the package'))
         outform.add_argument('--qf', "--queryformat", dest='queryformat',
                              default=QFORMAT_DEFAULT,
-                             help=_('format for displaying found packages'))
+                             help=_('format for showing found packages: '
```

Comment:
I'd prefer to keep the word "display" there. Ideally perhaps change it to:
> display format for listing packages:

Reply:
Fixed.